apis/v1alpha1/generator.yaml

```diff
@@ -13,6 +13,47 @@ operations:
       - Update
     resource_name: Rule
 resources:
+  Archive:
+    fields:
+      ArchiveName:
+        is_immutable: true
+        is_required: true
+      EventSourceARN:
+        is_immutable: true
+        is_required: true
+        references:
+          resource: EventBus
+          path: Status.ACKResourceMetadata.ARN
+    tags:
+      ignore: true
+    hooks:
+      sdk_create_post_set_output:
+        template_path: hooks/archive/sdk_create_post_set_output.go.tpl
+      sdk_update_pre_build_request:
+        template_path: hooks/archive/sdk_update_pre_build_request.go.tpl
+      sdk_update_post_build_request:
+        template_path: hooks/archive/sdk_update_post_build_request.go.tpl
+    print:
+      add_age_column: true
+      add_synced_column: false
+      additional_columns:
+        - name: ARN
+          json_path: .status.ackResourceMetadata.arn
+          type: string
+        - name: SYNCED
+          json_path: .status.conditions[?(@.type=="ACK.ResourceSynced")].status
+          type: string
```

Comment:
Did we fix this?

Reply:
Yes, it's even in the global `generator.yaml`. Let me rerun code-gen so this one gets updated.